tests/tests_system/test_process.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+import unittest
+
+from system.process import Process
+
+
+class TestProcess(unittest.TestCase):
+    def test(self):
+
+        process_handler = Process(".")
+
+        process_handler.start("ls", ".")
```

Comment:
What is this `ls` here for?

Reply:
This is just for spinning up a simple test process. It needs some commands to run.